charts/opentelemetry-collector/templates/NOTES.txt

```diff
@@ -50,6 +50,10 @@
 
 {{ end }}
 
+{{- if .Values.presets.coralogixExporter.pipeline }}
+[WARNING] The variable "presets.coralogixExporter.pipeline" is deprecated.  Use the variable "presets.coralogixExporter.pipelines" instead.
+{{- end }}
+
```

Comment:
Would also add a note to explain what happens when `pipeline` and `pipelines` are provided at the same time if we detect both are present.